presentation/frontend/src/presentation-frontend/PresentationManager.ts

```diff
@@ -278,7 +262,14 @@ export class PresentationManager implements IDisposable {
    * Function that is called when a new IModelConnection is used to retrieve data.
    * @internal
    */
-  public async onNewiModelConnection(_: IModelConnection) {}
+  public startiModelInitialization(_: IModelConnection) {}
+
+  /**
+   * Function that should be called to finish initialization that was started at {@link PresentationManager.startiModelInitialization}.
+   * Can be removed when {@link FavoritePropertiesManager.has} and {@link FavoritePropertiesManager.sortFields} are removed.
```

Comment:
APIs of the same package may be linked using double brackets

```suggestion
   * Function that should be called to finish initialization that was started at [[PresentationManager.startiModelInitialization]].
   * Can be removed when [[FavoritePropertiesManager.has]] and [[FavoritePropertiesManager.sortFields]] are removed.
```